### DISCUSSION TITLE
用一键脚本安装时候在安装 Ingress-nginx-controller 和 Kubeblocks.这步显示couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request

### DISCUSSION THREAD

**Discussion by lfhy** [labring/sealos#4527] [Q&A]
运行系统：Ubuntu 22.04.3 LTS
内存：32GB
根目录：100GB
内核：5.15.0

第一次部署时候就停在安装``` Ingress-nginx-controller ```和 ```Kubeblocks```这步 也是显示```couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request```

后续安装也停在这里
<img width="1289" alt="image" src="https://github.com/labring/sealos/assets/17682328/4d585d87-1aa1-4671-b5ac-294707262fd2">

iptable有加载
<img width="604" alt="image" src="https://github.com/labring/sealos/assets/17682328/53d5c068-ea27-4176-9c31-05f27b68e787">

**Comment by specialWinter** [comment]
你好 同样的问题 解决了嘛？遇到同样的问题
